containers/ddev-xhgui/var/www/xhgui/config/config.php

```diff
@@ -17,6 +17,7 @@
         'pass' => getenv('XHGUI_PDO_PASS') ?: 'db',
         'table' => getenv('XHGUI_PDO_TABLE') ?: 'results',
         'tableWatch' => getenv('XHGUI_PDO_TABLE_WATCHES') ?: 'watches',
+        'initSchema' => getenv('XHGUI_PDO_INIT_SCHEMA') ?: 'true',
```

Comment:
This is out of sync with upstream:

```
src/config.default.php:        'initSchema' => getenv('XHGUI_PDO_INITSCHEMA') ?: 'true',
```

It may be confusing to use different env var names

Reply:
Oh thanks, I didn't actually type, it's all autocomplete.